icons/zig.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Zig icon</title><path d="m16.627 19.372 2.98-3.45.785 2.352zm3.922-3.451V8.078h-1.412l1.569-1.412 1.412-2.196H24v14.902h-7.373l2.197-1.882.784-1.569zm-1.412-7.843 2.98-3.608-.156 2.824zM8.157 19.372.47 22.98 15.843 4.47l7.686-3.451zm10.039-3.451-2.51 1.412-.47 2.04H6.588V15.92zM8.784 4.47h8.628v3.451H5.804l2.98-1.882zm5.334 12.863 4.078-1.412-2.98 3.451zm-13.491.314 4.236-1.726-2.98 3.451zm5.177-9.726 2.98-3.451.941 2.196zm-.941 8-1.726.784-1.255 2.667H0V4.47h7.216l-2.04 1.726-.784 1.725h-.941v8zM2.98 5.725 7.216 4.47 4.392 7.921z"/></svg>
```

Comment:
I'm seeing a fair few [unnecessary paths](https://user-images.githubusercontent.com/1830867/116975185-241ff980-acb7-11eb-97f4-0f8d146000f5.png) inside this SVG. If you're able to remove those (Release, then Unite in Illustrator) then the icon otherwise matches up perfectly!